django/db/migrations/graph.py

```diff
@@ -84,6 +84,7 @@ class MigrationGraph:
     def __init__(self):
         self.node_map = {}
         self.nodes = {}
+        self.replaced_node_keys = set()
```

Comment:
`replaced_node_keys` is unused. Please revert changes in `django/db/migrations/graph.py`.